### PR TITLE
Implement Bridge between Cyphal/CAN and ROS for L3X-Z

### PR DESCRIPTION
**Pull request by aentinger**

### CODE REVIEW

**Comment by aentinger** [comment]
Hi @pavel-kirienko :coffee: :wave: 

It's not complete but you can see where the journey is going.

If you have ideas for a very quick implementation of storing bridge configuration in a config file and instantiating publisher/subscriptions using textual parameters only I'd love to hear it. Otherwise I'll go with this for the time being.

File: src/Node.cpp [hunk 1]
```diff
@@ -23,29 +23,79 @@ namespace l3xz
...
+  _node_hdl.setNodeId(get_parameter("can_node_id").as_int());
 
-  _can_mgr = std::unique_ptr<CanManager>
-    (new CanManager(get_logger(),
-                    get_parameter("can_iface").as_string(),
-                    [this](CanardFrame const & frame)
-                    {
-                      RCLCPP_INFO(get_logger(), "CAN frame received");
-                    }));
+  _can_mgr = std::unique_ptr<CanManager>(new CanManager(
```
Comment (pavel-kirienko) [inline src/Node.cpp:77]: This is a minor nit but static analyzers usually scream bloody murder when they see `new`, and for a good reason. I think a decent citizen would use a `std::make_unique` here or something like that.

**Review by pavel-kirienko — APPROVED** [review]
Okay this one seems fairly specialized for l-drei-xz. I sometimes wonder if there's a desire for a generalized bridge that just does the right thing automatically...

File: src/Node.cpp (hunk 1 above)
Comment (aentinger) [reply src/Node.cpp]: Huh, I did not know `std::make_unique` was a thing. Fixed in https://github.com/107-systems/ros2_cyphal_bridge/pull/45/commits/b4da6a3de553b5aab90be644bacd332c927b022f . Thank you for the input :bow: .

**Comment by aentinger** [comment]
> Okay this one seems fairly specialized for l-drei-xz. I sometimes wonder if there's a desire for a generalized bridge that just does the right thing automatically...

Yes, this one is fairly specialized for l-drei-xz. But I was wondering if you see a short road to a more generalized bridge which could be configured i.e. via a config file. However, if its a longer road I'd go with the specialized version until ENRICH.

**Pull request merged**